.github/workflows/build-and-release.yml

```
@@ -0,0 +1,141 @@
+          # Update pubspec.yaml
+          sed -i "s/^version: .*/version: ${FULL_VERSION}/" pubspec.yaml
+      
+      - name: Build APK (Android)
+        run: flutter build apk --release
+      
+      - name: Build App Bundle (Android)
+        run: flutter build appbundle --release
+      
+      - name: Commit version bump
+        if: github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master')
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2015:info:4:18: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)